src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/settings/SettingsViewFacade.java

```diff
@@ -2,117 +2,149 @@
+import edu.kit.pse.osip.monitoring.controller.SettingsViewInterface;
 
 /**
  * Provides a single access point to the user-set settings.
+ * 
+ * @author Martin Armbruster
+ * @version 1.0
  */
-public class SettingsViewFacade implements edu.kit.pse.osip.monitoring.controller.SettingsViewInterface {
+public final class SettingsViewFacade implements SettingsViewInterface {
     /**
-     * The current settings view containg the user-set settings.
+     * The current settings view contaning the user-set settings.
      */
     private SettingsMainWindow currentSettingsWindow;
+    
     /**
      * Shows the settings view with the current settings.
+     * 
      * @param currentSettings The current settings used for displaying them.
      */
-    public final void showSettingsWindow (ClientSettingsWrapper currentSettings) {
+    public void showSettingsWindow(ClientSettingsWrapper currentSettings) {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Sets the handler for the cancel button in the settings view.
+     * 
      * @param handler The handler for the cancel button in the settings view.
      */
-    public final void setSettingsCancelButtonHandler (SettingsCancelButtonHandler handler) {
+    public void setSettingsCancelButtonHandler(AbstractSettingsCancelButtonHandler handler) {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Sets the handler for the save button in the settings view.
+     * 
      * @param handler The handler for the save button in the settings view.
      */
-    public final void setSettingsSaveButtonHandler (SettingsSaveButtonHandler handler) {
+    public void setSettingsSaveButtonHandler(AbstractSettingsSaveButtonHandler handler) {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Hides the settings view.
      */
-    public final void hideSettingsWindow () {
+    public void hideSettingsWindow() {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Returns the host name for the servers.
+     * 
      * @return the host name for the servers.
      */
-    public final String getServerHostname () {
+    public String getServerHostname() {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Returns the port number for a specified server.
-     * @return
-     * the port number of a specified server.
+     * 
+     * @return the port number of a specified server.
      * @param serverNumber Number of the server whose port number will be returned.
      */
-    public final int getServerPort (int serverNumber) {
+    public int getServerPort(int serverNumber) {
         throw new RuntimeException("Not implemented!");
     }
+    
     /**
      * Returns the update time interval.
+     * 
      * @return the update time interval.
      */
-    public final int getTime () {
+    public int getTime() {
```

Comment:
Bitte umbenennen in `getUpdateInterval()`. `Date.getTime()` ist was komplett anderes aber die Funktion heißt zu ähnlich